### DISCUSSION TITLE
Mapping keycloak groups to cluster roles

### DISCUSSION THREAD

**Discussion by leewrigh** [tchiotludo/akhq#1859] [Q&A]
Does anyone have an example of the keycloak group mapper changes needed for the 0.25+ versions?
Looks like groups need to be a map but the default keycloak group mapper will just provide a list of groups (with paths)

thanks in advance

**Comment by AlexisSouquiere** [comment]
Can you give more details (like Keycloak setup for version 0.24.0) ? Are you in this use case trying to build the claim with the groups field ? https://akhq.io/docs/configuration/authentifications/oidc.html#direct-oidc-mapping
